microsoft-edge/webview2/release-notes/index.md

```diff
@@ -85,6 +85,237 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
+   * [ICoreWebView2EnvironmentOptions7::get_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7?view=webview2-1.0.2470-prerelease&preserve-view=true#get_releasechannels)
+   * [ICoreWebView2EnvironmentOptions7::put_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7?view=webview2-1.0.2470-prerelease&preserve-view=true#put_releasechannels)
+
+* [COREWEBVIEW2_CHANNEL_SEARCH_KIND enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.2470-prerelease&preserve-view=true#corewebview2_channel_search_kind)
+
+* [COREWEBVIEW2_RELEASE_CHANNELS enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.2470-prerelease&preserve-view=true#corewebview2_release_channels)
+   * `COREWEBVIEW2_RELEASE_CHANNELS_NONE`
+   * `COREWEBVIEW2_RELEASE_CHANNELS_STABLE`
+   * `COREWEBVIEW2_RELEASE_CHANNELS_BETA`
+   * `COREWEBVIEW2_RELEASE_CHANNELS_DEV`
+   * `COREWEBVIEW2_RELEASE_CHANNELS_CANARY`
+
+---
+
+
+<!-- ---------- -->
+* Added the `FailureSourceModulePath` property to the `ProcessFailedEventArgs` type, to specify the full path of the module that caused the crash in cases of Windows code integrity failures - that is, when a process exited with `STATUS_INVALID_IMAGE_HASH`.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2ProcessFailedEventArgs` Class:
+   * [CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs.failuresourcemodulepath?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2ProcessFailedEventArgs` Class:
+   * [CoreWebView2ProcessFailedEventArgs.FailureSourceModulePath Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs?view=webview2-winrt-1.0.2470-prerelease&preserve-view=true#failuresourcemodulepath)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ProcessFailedEventArgs3](/microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs3?view=webview2-1.0.2470-prerelease&preserve-view=true)
+   * [ICoreWebView2ProcessFailedEventArgs3::get_FailureSourceModulePath](/microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs3?view=webview2-1.0.2470-prerelease&preserve-view=true#get_failuresourcemodulepath)<!--no put-->
+
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+###### Runtime-only
+
+* Fixed a reliability regression that could crash the application process when an old version of WebView2 client DLL is unloaded.
+
+
+###### Runtime and SDK
+
+* Ensured that the WebView2 temporary download folder is unique per user data folder, and doesn't interfere with other apps or the browser.
+<!-- todo: confirm that this is both Runtime & SDK.  vs. the orig commit that added this item in both the "Runtime-only" and "SDK-only" sections -->
```

Comment:
confirm that this is both Runtime & SDK. vs. the orig [commit](https://github.com/MicrosoftDocs/edge-developer/pull/3095/commits/ae1229a86d2f2fee92b51ee1f48568966fd14c70) that added this item in both the "Runtime-only" and "SDK-only" sections:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/e7e5edde-4f12-4588-81fe-dbc63bff8302)
@peiche-jessica @victorhuangwq

Reply:
I checked the original work item. This should be runtime-only. Item owner is Diana if we'd like triple check.

Reply:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/1c8e4392-2e2d-4248-a1c9-aec6ada817d2)